inference-engine/tests_deprecated/functional/shared_tests/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(TARGET_NAME IESharedTests)
 
 list(APPEND SHARED_LIBRARIES
         ngraph
+        ngraph::frontend_manager
```

Comment:
Why do we need `frontend_manager` in deprecated tests?

Reply:
reverted. kmb legacy tests keep failing because of lacking thee dependency